templates/email/template/adherent_account_activation_message.html.twig

```diff
@@ -0,0 +1,35 @@
+{% block subject %}
+    Plus qu'une étape !
+{% endblock %}
+
+{% block body_html %}
+
+    {% embed 'email/layout.html.twig' %}
+        {% block content %}
+            {% verbatim %}
+            <tr>
+              <td align="left">
+                  <p>
+                      Bonjour {{var:target_firstname:""}},
+                  </p>
+                  <p>
+                      Merci d'avoir rejoint le mouvement !
+                  </p>
+                  <p>
+                      Il ne vous reste plus qu’une étape : confirmer  votre adresse e-mail.
+                  </p>
+                </div>
```

Comment:
this ending div without opening div breaks email structure in gmail.
![screen shot 2017-10-16 at 17 28 33](https://user-images.githubusercontent.com/9162276/31620412-7ae30a46-b297-11e7-9630-4eb11472cdce.png)

Reply:
@marineb could you check?